lighthouse-core/lib/i18n/i18n.js

```diff
@@ -217,28 +209,34 @@ function _processParsedElements(icuMessage, argumentElements, values = {}) {
     // Format values for known styles.
     if (format.style === 'milliseconds') {
       // Round all milliseconds to the nearest 10.
-      values[id] = Math.round(value / 10) * 10;
+      formattedValues[id] = Math.round(value / 10) * 10;
     } else if (format.style === 'seconds' && id === 'timeInMs') {
       // Convert all seconds to the correct unit (currently only for `timeInMs`).
-      values[id] = Math.round(value / 100) / 10;
+      formattedValues[id] = Math.round(value / 100) / 10;
     } else if (format.style === 'bytes') {
       // Replace all the bytes with KB.
-      values[id] = value / 1024;
+      formattedValues[id] = value / 1024;
+    } else {
+      // All other numbers passed through unchanged.
```

Comment:
```suggestion
      // All other numbers (e.g. ∞) passed through unchanged.
```

Reply:
Infinity could be passed to the other styles as well (the issue in #9684 is passing it through `bytes`, it's just that `Infinity / 1024 === Infinity` :)

Reply:
tried to make this comment clearer as well

Reply:
tried to make this comment clearer as well